loader/directory_refresher.go

```diff
@@ -3,7 +3,14 @@ package loader
 import "path/filepath"
 
 type DirectoryRefresher struct {
-	currDir string
+	currDir  string
+	watchOps map[FileSystemOp]struct{}
```

Comment:
Given this is unexported, how are users expected to be able to manipulate this? 

A couple options:
1) export it and allow users to set it directly
2) create a new method to allow setting it (similar to WatchDirectory)

I tend to lean towards 1, but acknowledge this type already uses approach 2. Either should be okay.

Reply:
Totally forgot to add this method as I never changed the Refresher object after creation. I added the WatchFileSystemOps method and a small unit test.